frappe/core/doctype/file_extension/file_extension.json

```diff
@@ -0,0 +1,46 @@
+{
+    "actions": [],
+    "allow_rename": 1,
+    "autoname": "field:name1",
+    "creation": "2023-09-20 17:48:47.905037",
+    "default_view": "List",
+    "doctype": "DocType",
+    "editable_grid": 1,
+    "engine": "InnoDB",
+    "field_order": [
+     "name1"
+    ],
+    "fields": [
+     {
+      "fieldname": "name1",
```

Comment:
This is a weird fieldname, please keep them descriptive and human-readable wherever possible.